client/src/pages/manager/login/login.css

```diff
@@ -0,0 +1,31 @@
+.login-container {
+	@apply flex items-center justify-center w-full h-screen;
+}
+
+.login-box {
+	@apply flex flex-col justify-center items-center w-[420px] h-[360px] m-auto rounded-xl drop-shadow-2xl bg-[#161b22];
+}
+
+.login-title {
+	@apply my-2 text-3xl text-center text-white font-bold;
+}
+
+.login-form {
+	@apply flex flex-col justify-center items-center w-full h-auto my-2;
+}
+
+.login-input {
```

Comment:
There is an unpleasant blue and white border when you click on the email form.

<img width="408" alt="Screen Shot 2022-11-03 at 6 33 42 PM" src="https://user-images.githubusercontent.com/36748732/199866188-1e9cbf43-7dc2-40c3-9dd1-01bf6a8651aa.png">

Recommend implementing our own border when focusing using similar styling (color probably should be something else):
focus:border-blue-600 focus:outline-none

<img width="356" alt="Screen Shot 2022-11-03 at 6 35 34 PM" src="https://user-images.githubusercontent.com/36748732/199866424-7e03db7d-9612-4196-a06c-1b4b65491d86.png">

Reply:
I changed the border to white when focusing. It looks better.